src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -25,8 +22,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session);
 
     if (!hasFetchedBoAndMoData(appData)) {
-      await retrieveBeneficialOwners(req, appData);
-      await retrieveManagingOfficers(req, appData);
+      await getCompanyOfficers(req, appData[EntityNumberKey] as string);
```

Comment:
This will still need to be replaced by map methods rather than await the service call itself?

Reply:
yeah I agree with @olaych, I think keeping the methods retrieveManagingOfficers and retrieveBeneficialOwners is good as mapping stuff will need to add them back in anyways.

Reply:
Let's leave that to the mappings. I did not want two one line functions.